openmc/settings.py

```diff
@@ -817,52 +821,52 @@ def resonance_scattering(self, res):
         self._resonance_scattering = res
 
     @volume_calculations.setter
-    def volume_calculations(self, vol_calcs):
+    def volume_calculations(self, vol_calcs: Union[VolumeCalculation, typing.Iterable[VolumeCalculation]]):
```

Comment:
This is definitely stretching much past 90 characters. Can you wrap this line? (yes, yes, I know black would have fixed this all for you :stuck_out_tongue_closed_eyes:)

Reply:
I love it when a plan comes together :smile: 
Yes certain I can wrap this line manually.

Reply:
black suggests changing in this way

from this
```python
    def volume_calculations(self, vol_calcs: Union[VolumeCalculation, typing.Iterable[VolumeCalculation]]):
```

to this
```python
    def volume_calculations(
        self, vol_calcs: Union[VolumeCalculation, typing.Iterable[VolumeCalculation]]
    ):
```

Which ends up with 86 characters, so I've wrapped it in this way but I can reduce it further if required as some people like an 80 char limit